app/containers/CreateWallet/CreateWallet.jsx

```diff
@@ -114,15 +124,15 @@ export default class CreateWallet extends React.Component<Props, State> {
               />
               <PasswordInput
                 label="Passphrase"
-                value={passphrase}
-                onChange={e => this.setState({ passphrase: e.target.value })}
+                onChange={this.handleChangePassphrase}
                 placeholder="Password"
+                error={this.state.passphraseError}
```

Comment:
super minor, since we are destructing state on line `71`, could move `passphraseError` there too for consistency?